manifests/testing/kubevirt-testing-infra.yaml

```diff
@@ -124,6 +124,7 @@ spec:
           hostPath:
             path: /mnt/local-storage
             type: DirectoryOrCreate
+---
```

Comment:
thanks for catching that! :+1:

Reply:
This only became an issue when I've added the `oc wait pods -l "kubevirt.io=disks-images-provider"` in `test-cnv.sh`.